plugins/VstBase/RemoteVstPlugin.cpp

```diff
@@ -2301,7 +2300,7 @@ void RemoteVstPlugin::guiEventLoop()
 		{
 			XNextEvent(m_display, &e);
 		
-			if (e.type == ClientMessage && e.xclient.data.l[0] == m_wmDeleteMessage)
+			if (e.type == ClientMessage && static_cast<Atom>(e.xclient.data.l[0]) == m_wmDeleteMessage)
```

Comment:
Don't we use e here?

Reply:
That's a different `e` in a different function.

Reply:
Oops mb